src/components/common/buttons/ShareButton.js

```diff
@@ -13,21 +14,25 @@ type ShareButtonProps = {
   buttonProps: any,
 }
 
+const log = logger.child({ from: 'ShareButton' })
+
 const ShareButton = ({ share, onPressDone, actionText, ...buttonProps }: ShareButtonProps) => {
   const [showErrorDialog] = useErrorDialog()
+  const { canShare } = useNativeSharing()
+
+  log.info('getPaymentLink', { share })
```

Comment:
don't log directly in render, use effects

Reply:
done